application/backend/src/main/resources/apps/bulma/components/tabs/tab/tab.html

```diff
@@ -0,0 +1,72 @@
+<!--/*
+  ~
+  ~     Copyright (C) 2022 Dynamic Solutions
+  ~
+  ~     Licensed under the Apache License, Version 2.0 (the "License");
+  ~     you may not use this file except in compliance with the License.
+  ~     You may obtain a copy of the License at
+  ~
+  ~         http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~     Unless required by applicable law or agreed to in writing, software
+  ~     distributed under the License is distributed on an "AS IS" BASIS,
+  ~     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~     See the License for the specific language governing permissions and
+  ~     limitations under the License.
+  ~
+*/-->
+<sly data-sly-use.model="pl.ds.bulma.components.models.TabComponent">
+    <sly data-sly-test="${model.content}">
+        <!--        edit mode layout-->
+        <sly data-sly-test="${wcmmode.isEdit}">
+            <div style="margin-bottom: 20px; border-style: dashed; border-color: #a8a4a4;">
+                <sly data-sly-test.parent="${resource.parent}">
+                    <div class="tabs ${parent.size ? parent.size : ''} ${parent.alignment ? parent.alignment : ''} ${parent.boxed ? 'is-boxed' : ''} ${parent.fullWidth ? 'is-fullwidth' : ''} ${parent.toggle ? 'is-toggle' : ''} ${parent.toggleRounded ? 'is-toggle-rounded' : ''}">
```

Comment:
A lot of these conditions in one line. Maybe it can be simplified somehow by referring to the list of classes prepared in the model? Something like ${model.tabClasses @ join=' '} ?